core/save-profiledata.cpp

```diff
@@ -45,6 +47,16 @@ static std::string video_time(int secs)
 	return format_string_std("%d:%02d:%02d.000,", hours, mins, secs);
 }
 
+void replace_all(std::string& str, const std::string& old_value, const std::string& new_value) {
```

Comment:
I know it's a "religious" issue, but as per coding-style, we put the`&` with the variable, not the type. Rationale: That's how the C++ grammar works, like it or not.

Also I suggest making this a global helper function.

If not a global helper, it should be of static linkage.

Reply:
Fixed. Made the function static.